Modules/xASL_module_Structural.m

```diff
@@ -225,14 +225,6 @@
 		end
 
 		if xASL_exist(x.P.Path_T1c, 'file') || xASL_exist(x.P.Path_T2, 'file')
```

Comment:
Why not moving this check to the beginning of `xASL_wrp_LinearReg_Others2T1w` and removing it here?

Reply:
Good idea. Done